Account/FxAClient10.swift

```diff
@@ -142,13 +142,17 @@ open class FxAClient10 {
      * lowercase-hex-encoded first 16 bytes of the SHA-256 hash of the
      * bytes of kB.
      */
-    open class func computeClientState(_ kB: Data) -> String? {
-        if kB.count != 32 {
-            return nil
-        }
+    open class func computeClientState(_ kB: Data) -> String {
         return kB.sha256.subdata(in: 0..<16).hexEncodedString
     }
 
+    open class func deriveKSync(_ kB: Data) -> Data {
+        let salt = Data()
```

Comment:
Is this to generate random bytes? Is there a more explicit way to do this swift I wonder.